projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -258,6 +260,19 @@ export default function SubscriptionControls( {
 						setAttributes( { buttonOnNewLine: ! buttonOnNewLine } );
 					} }
 				/>
+
+				<TextareaControl
+					value={ subscribePlaceholder }
+					label={ __( 'Input placeholder text', 'jetpack' ) }
+					help={ __( 'Edit the placeholder text of the email address input.', 'jetpack' ) }
+					onChange={ placeholder => setAttributes( { subscribePlaceholder: placeholder } ) }
+				/>
+				<TextareaControl
+					value={ submitButtonText }
+					label={ __( 'Submit button label', 'jetpack' ) }
+					help={ __( 'Edit the label of the button a user clicks to subscribe.', 'jetpack' ) }
+					onChange={ text => setAttributes( { submitButtonText: text } ) }
+				/>
```

Comment:
Do we need this, given that one can click on the button to edit its text?

Reply:
ah, I suppose that makes sense.

It looks like the sidebar controls over less flexibility with line breaks though?


https://github.com/Automattic/jetpack/assets/426388/8fef78fc-728a-4e83-be07-01966aab8fd4

Reply:
I don't necessarily see this as an issue, especially since a multi-line button would visually clash with the single-line input in this specific case.

But I don't mind looking into it. Maybe in a separate PR so that we can merge this one?